vizro-ai/docs/pages/explanation/why-use-vizro-ai.md

```diff
@@ -0,0 +1,12 @@
+# Why use Vizro-AI over ChatGPT?
+
+Vizro-AI offers an easy way to turn natural language instructions into professional visualizations, like interactive charts and dashboards, without needing advanced coding skills.
+
+Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
+
+- **Tailored for data visualization**: Vizro-AI is specifically designed to create and design charts and dashboards.
+- **Enforced design best practices automatically**: Using Vizro’s themes, your visualizations are built with best practices in mind from the start.
+- **Prompt optimization**: Vizro-AI uses advanced prompt engineering to incorporate additional context and data-specific details, making your visualizations more accurate.
```

Comment:
This seems too focused on the technical implementation rather than the user benefit. As a user, I'm more interested in how it provides data insights that complement my charts. So I think the emphasis should be on the added value rather than the technical details like prompt optimization?  Maybe something like below?

```suggestion
- **Data Insights**: Vizro-AI leverages advanced prompt engineering to integrate additional context and data-specific insights, enriching your visualizations with more comprehensive information and allowing for more effective data storytelling.
```